From 3be551cf007c3740d017a7a301f6bbcfb0c72d74 Mon Sep 17 00:00:00 2001
From: NaveenVenturi1203 <venturi.naveen@intel.com>
Date: Mon, 18 Nov 2024 06:41:10 +0000
Subject: [PATCH] Change Lens Facing to External for External Camera HAL

Issue Detailed: Lens Facing was set to back with support for external
camera  leading to CTS Failure as CTS was expecting camera with lens
facing external

Issue Fixed: Changed the Lens Facing Metadata to EXTERNAL for External
Camera HAL

Tested-On: CTS Test Cases Passed

Tracked-On: OAM-127055

Signed-off-by: NaveenVenturi1203 <venturi.naveen@intel.com>
---
 camera/device/default/ExternalCameraDevice.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/camera/device/default/ExternalCameraDevice.cpp b/camera/device/default/ExternalCameraDevice.cpp
index 50fbdd4efe..8e8474800f 100644
--- a/camera/device/default/ExternalCameraDevice.cpp
+++ b/camera/device/default/ExternalCameraDevice.cpp
@@ -420,7 +420,7 @@ status_t ExternalCameraDevice::initDefaultCharsKeys(
     const uint8_t opticalStabilizationMode = ANDROID_LENS_OPTICAL_STABILIZATION_MODE_OFF;
     UPDATE(ANDROID_LENS_INFO_AVAILABLE_OPTICAL_STABILIZATION, &opticalStabilizationMode, 1);
 
-    const uint8_t facing = ANDROID_LENS_FACING_BACK;
+    const uint8_t facing = ANDROID_LENS_FACING_EXTERNAL;
     UPDATE(ANDROID_LENS_FACING, &facing, 1);
 
     // android.noiseReduction
-- 
2.34.1

